apps/storybook/src/DomainWidget.stories.tsx

```diff
@@ -4,12 +4,12 @@ import { COLOR_SCALE_TYPES } from '@h5web/shared';
 import type { Meta, StoryObj } from '@storybook/react';
 import { useState } from 'react';
 
-import Center from './decorators/Center';
+import DomainWidgetDecorator from './decorators/DomainWidgetDecorator';
 
 const meta = {
   title: 'Toolbar/DomainWidget',
   component: DomainWidget,
-  decorators: [Center],
+  decorators: [DomainWidgetDecorator],
```

Comment:
I noticed the domain widget stories required a minimum height for the domain tooltip to be usable. Instead of adding `min-height` to `Center`, which would have made it less generic, I decided to turn `Center` into a decorator dedicated to `DomainWidget` and stopped using it for the `Histogram` stories, which work fine without it.